nss-evening-cohort-14/whalom-park-dashboard-whalom-park-dashboard
# Mg charging visitors

## Description
Built functionality for visitors to pay for a ride or vendor, and to keep a log of that data. Only available when logged on.

## Related Issue

#84 
#88 

## Motivation and Context
Want to keep track of what is selling at our park and monitor performance of our attractions.

## How Can This Be Tested?

git fetch mg-charging-visitors
Log on, click the Passing Time button found on any page, monitor Firebase for new Log data or see the Performance page for updated chart data.

## Screenshots (if appropriate):

## Types of changes
